lib/websql/WebSQLDatabase.js

```diff
@@ -14,6 +14,22 @@ var COMMIT = [
   {sql: 'END;', args: []}
 ];
 
+var createInfoTable = {
+  sql: 'CREATE TABLE IF NOT EXISTS __WebKitDatabaseInfoTable__ ' +
+    '(key TEXT NOT NULL ON CONFLICT FAIL UNIQUE ON CONFLICT REPLACE,' +
+    'value TEXT NOT NULL ON CONFLICT FAIL)',
```

Comment:
What is the point of `NOT NULL ON CONFLICT FAIL`? As I understand it from [the SQLite docs](https://sqlite.org/lang_conflict.html), this just says that if you try to insert null `TEXT`, it will fail, which I believe is the same thing that happens by default?